math/factorial_recursion.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+
+#define MAX_LIMIT 200 
+
+unsigned long long factResults[MAX_LIMIT] = {0};
+
+unsigned long long computeFactorial(int sequenceNumber) {

```

Comment:
using lengthy data types is not recommended 
use the _t types from stdint.h
```c
#include<stdint.h>

uint64_t computeFactorial(...)
```
```suggestion
uint64_t computeFactorial(int sequenceNumber) {

```